beacon_node/execution_layer/src/lib.rs

```diff
@@ -2052,67 +2065,127 @@ impl From<serde_json::Error> for BlobTxConversionError {
-            .max_priority_fee_per_gas
-            .ok_or(BlobTxConversionError::MaxPriorityFeePerGasMissing)?;
-        let max_fee_per_gas = transaction
-            .max_fee_per_gas
-            .ok_or(BlobTxConversionError::MaxFeePerGasMissing)?;
-        let gas = if transaction.gas > Uint256::from(u64::MAX) {
+
+        // maxPriorityFeePerGas
+        let max_priority_fee_per_gas =
+            max_priority_fee_per_gas.ok_or(BlobTxConversionError::MaxPriorityFeePerGasMissing)?;
+
+        // maxFeePerGas
+        let max_fee_per_gas = max_fee_per_gas.ok_or(BlobTxConversionError::MaxFeePerGasMissing)?;
+
+        // gas
+        let gas = if gas > Uint256::from(u64::MAX) {
             return Err(BlobTxConversionError::GasTooHigh);
         } else {
-            transaction.gas.as_u64()
+            gas.as_u64()
         };
-        let to = transaction.to;
-        let value = transaction.value;
-        let data = VariableList::new(transaction.input.to_vec())?;
+
+        // data (a.k.a input)
+        let data = VariableList::new(input.to_vec())?;
+
+        // accessList
         let access_list = VariableList::new(
-            transaction
-                .access_list
-                .as_ref()
+            access_list
                 .ok_or(BlobTxConversionError::AccessListMissing)?
                 .0
-                .iter()
+                .into_iter()
                 .map(|access_tuple| {
+                    let AccessListItem {
+                        address,
+                        storage_keys,
+                    } = access_tuple;
                     Ok(AccessTuple {
-                        address: access_tuple.address,
-                        storage_keys: VariableList::new(access_tuple.storage_keys.clone())?,
+                        address,
+                        storage_keys: VariableList::new(storage_keys)?,
                     })
                 })
                 .collect::<Result<Vec<AccessTuple>, BlobTxConversionError>>()?,
         )?;
-        let max_fee_per_data_gas = transaction
-            .other
-            .get("maxFeePerDataGas")
-            .ok_or(BlobTxConversionError::MaxFeePerDataGasMissing)?
-            .as_str()
-            .ok_or(BlobTxConversionError::MaxFeePerDataGasMissing)?
-            .parse()
-            .map_err(|_| BlobTxConversionError::MaxFeePerDataGasMissing)?;
-        let blob_versioned_hashes = serde_json::from_str(
-            transaction
-                .other
-                .get("blobVersionedHashes")
-                .ok_or(BlobTxConversionError::BlobVersionedHashesMissing)?
+
+        // ******************** BlobTransaction `other` fields ********************
+        //
+        // Here we use the `other` field in the `ethers-rs` `Transaction` type because
+        // `ethers-rs` does not yet support SSZ and therefore the blobs transaction type.
+
+        // maxFeePerDataGas
+        let data_gas_bytes = eth2_serde_utils::hex::decode(
```

Comment:
We could also convert to a U256 directly from the string with this method https://docs.rs/ethereum-types/latest/ethereum_types/struct.U256.html#method.from_str_radix

I couldn't find documentation that the EL represents this field as a hex string in the json, so assuming you found the type while testing it out.

Reply:
yep the `json_snooper` helped me out here